queries.sql

```diff
@@ -1,11 +1,67 @@
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals WHERE name LIKE '%mon%';
-SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-12-31' AND '2019-01-01';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
 SELECT * FROM animals WHERE neutered = 't' AND escape_attempts < 3;
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals
-SELECT * FROM animals WHERE neutered = 't';
\ No newline at end of file
+SELECT * FROM animals WHERE neutered = 't';
+
+/* Rollback practise */
+ BEGIN;
+ UPDATE animals SET species = 'unspecified';
+ ROLLBACK;
+
+/* Add species data as transaction */
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+COMMIT;
+
```

Comment:
- [ ] To verify that the changes persist after changes, please 🙏🏽 add this line 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/215282897-4332a9de-10f5-4c3b-9eb4-b33629ac91fe.png)